lib/net/pool.js

```diff
@@ -3157,6 +3157,7 @@ class Pool extends EventEmitter {
   async handleGetProof(peer, packet) {
     const {root, key} = packet;
     const proof = await this.chain.db.prove(root, key);
+    peer.increaseDos(1, Date.now());
```

Comment:
Thoughts on using `Date.now` vs `util.ms`?

`util.ms` simply wraps `Date.now`.

I'm not particularly opinionated on either, it looks like `Date.now` is used more often so maybe its better to stick with that.